docs/code/getting-started.md

```diff
@@ -417,54 +378,108 @@ Let's review all we've learned to do:
 
 # Adding Translations {#Adding-Translations}
-Now that you've learned about CMake, the next step is to make your app able to be translated to different languages. The first thing you need to know is how to convert strings in your code into translatable strings. Here's an example:
+Now that you've learned about Meson, the next step is to make your app able to be translated to different languages. The first thing you need to know is how to convert strings in your code into translatable strings. Here's an example:
 
         stdout.printf ("Not Translatable string");
         stdout.printf (_("Translatable string!"));
 
         string normal = "Another non-translatable string";
         string translated = _("Another translatable string");
 
-See the difference? We just added `_()` around the string! Well, that was easy!
+See the difference? We just added `_()` around the string! Well, that was easy! Go back to your project and make all your strings translatable by adding `_()`.
+
+Now we have to make some changes to our Meson build system and add a couple new files to describe which files we want to translate and which languages we want to translate into.
 
-1. Go back to your project and make all your strings translatable by adding `_()`
+1. Open up your "meson.build" build and add these lines below your project declaration:
 
-2. Add the following lines in the "CMakeLists.txt" file you created a moment ago:
+        # Include the translations module
+        i18n = import('i18n')
 
-        # Translation files
-        set (GETTEXT_PACKAGE "${CMAKE_PROJECT_NAME}")
-        add_definitions (-DGETTEXT_PACKAGE=\"${GETTEXT_PACKAGE}\")
-        add_subdirectory (po)
+        # Set our translation domain
+        add_global_arguments('-DGETTEXT_PACKAGE="@0@"'.format (meson.project_name()), language:'c')
 
-3. Create a directory named "po" on the root folder of your project. Inside of your po directory you will need to create another CMakeLists.txt file. This time, it's contents will be:
+2. Remove the lines that install your .desktop and appdata files and replace them with the following:
 
-        include (Translations)
-            add_translations_directory(${GETTEXT_PACKAGE})
-            add_translations_catalog(${GETTEXT_PACKAGE}
-            ../src/
+        #Translate and install our .desktop file
+        i18n.merge_file(
+            input: meson.project_name() + '.desktop.in',
+            output: meson.project_name() + '.desktop',
+            po_dir: join_paths(meson.source_root(), 'po'),
+            type: 'desktop',
+            install: true,
+            install_dir: join_paths(get_option('datadir'), 'applications')
         )
 
-4. On your build directory execute the following commands:
+        #Translate and install our .appdata file
+        i18n.merge_file(
+            input: meson.project_name() + '.appdata.xml.in',
```

Comment:
Line 420:
I suppose `input: join_paths('data', meson.project_name() + '.appdata.xml.in'),` is correct.